src/demo.ts

```diff
@@ -310,4 +372,42 @@ export async function mount({ element, init, listeners }: MountOptions) {
 			endColumn,
 		};
 	}
+
+	/**
+	 * Create quick-fix code action.
+	 */
+	function createQuickFixCodeAction(
+		title: string,
+		marker: editor.IMarkerData,
+		model: editor.ITextModel,
+		fix: stylelint.EditInfo,
+	): languages.CodeAction {
+		const start = model.getPositionAt(fix.range[0]);
+		const end = model.getPositionAt(fix.range[1]);
```

Comment:
Awesome to see how this integrates.
Makes me feel confident that we are on the right track.

Thank you so much for working on this @ota-meshi 🙇